addons/account/tests/test_account_account.py

```diff
@@ -32,7 +31,6 @@ def test_account_company(self):
         account = self.env['account.account'].create({
             'code': '180001',
             'name': 'My Account in Company 2',
-            'account_type': 'asset_current',
```

Comment:
Why?

Reply:
It's not necessary since the compute automatically gives an account type to the new account